examples/image_border.py

```diff
@@ -0,0 +1,42 @@
+"""
+Layer Border Overlay
+====================
+
+Display an image in napari and add a border overlay (i.e. `bounding_box`).
+
+Demonstrates visualization of many border overlay properties,
+including line color, line thickness, point size, opacity.
+In addition, this viewer shows how layer overlays interact when draw on top of each other (at the intersection of the grid).
+Use `viewer.layers[0].bounding_box` to view all modifiable attributes of the border overlay.
+
+For an example showing how bounding box extents are visualized, see
+:ref:`sphx_glr_gallery_layer_bounding_box.py`.
+
+.. tags:: visualization-advanced
+"""
+
+from skimage import data
+
+import napari
+
+# Create a napari viewer with a 3D, 2 channel image, with the `view_image` convenience function.
+viewer = napari.view_image(
+    data.cells3d(), channel_axis=1, name=['membrane', 'nuclei']
+)
+viewer.grid.enabled = True
+
+# Add a border overlay to each layer, and modify properties of the border overlay.
+viewer.layers[0].bounding_box.visible = True
+viewer.layers[0].bounding_box.line_color = 'cyan' # default: 'red'
+viewer.layers[0].bounding_box.line_thickness = 5 # default: 1, apparent max: 5; GPU-dependent, see: https://vispy.org/api/vispy.scene.visuals.html#vispy.scene.visuals.Line
```

Comment:
```suggestion
viewer.layers[0].bounding_box.line_thickness = 5 # default: 1, maximum is GPU-dependent, see: https://vispy.org/api/vispy.scene.visuals.html#vispy.scene.visuals.Line
```
It's 10 for me 🤷 
Lets just say it's GPU dependent and leave it at that.